UploadDaemon/Report/Testwise/Test.cs

```diff
@@ -75,7 +75,12 @@ public Test Union(Test other)
 
         public SimpleCoverageReport ToSimpleCoverageReport()
         {
-            return new SimpleCoverageReport(CoverageByPath.First().Files.ToDictionary(file => file.FileName, file => (FileCoverage)file));
+            IDictionary<string, FileCoverage> lineCoverageByFile = new Dictionary<string, FileCoverage>();
```

Comment:
The Naming is a bit confusing here for me. What is the difference between Path and File? I can see that they are different things from the data structures behind it but I don't know why? Could you change the names or add some comments to make it easier to understand?

Reply:
I agree, changing to path